### ISSUE TITLE
Villagers in tavern stand on chairs instead of sitting

### ISSUE THREAD

**Issue by IridescentShadow** [TangoTek/TekTopia-Community#127]
**Describe the bug**
Villagers will sit in chairs in their homes, but always stand on the ones in the tavern

**To Reproduce**
1. Go to the tavern
2. Watch villagers stand on chairs instead of sitting

**Expected behavior**
Villagers should sit in their chairs at the tavern

**Screenshots**
![2019-05-27_21 13 52](https://user-images.githubusercontent.com/7838663/58493294-9b01fe80-8140-11e9-948d-a3852cc61da9.png)

** Server/Location **
SP, but can share world file if needed

**Comment by mrhyde1st** [comment]
Maybe they watched Dead Poet's Society "Oh Captain My Captain!"

**Comment by IridescentShadow** [comment]
I finally solved this. The issue was the marker for the tavern being next to the door at the rear of my screenshot that was one block higher with stairs coming down. Someone in Discord mentioned that items capable of being interacted with by a villager (chairs, beds, crafting tables, furnaces, etc) need to be on the same level as the door with the marker. As soon as I moved the marker, they started sitting.
